libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/AudioCapabilities.java

```diff
@@ -41,27 +41,27 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 import com.google.common.primitives.Ints;
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 
 /** Represents the set of audio formats that a device is capable of playing. */
 @UnstableApi
 public final class AudioCapabilities {
 
-  private static final int DEFAULT_MAX_CHANNEL_COUNT = 8;
+  private static final int DEFAULT_MAX_CHANNEL_COUNT = 10;
   private static final int DEFAULT_SAMPLE_RATE_HZ = 48_000;
 
   /** The minimum audio capabilities supported by all devices. */
   public static final AudioCapabilities DEFAULT_AUDIO_CAPABILITIES =
       new AudioCapabilities(new int[] {AudioFormat.ENCODING_PCM_16BIT}, DEFAULT_MAX_CHANNEL_COUNT);
 
-  /** Audio capabilities when the device specifies external surround sound. */
-  @SuppressWarnings("InlinedApi")
-  private static final AudioCapabilities EXTERNAL_SURROUND_SOUND_CAPABILITIES =
-      new AudioCapabilities(
-          new int[] {
-            AudioFormat.ENCODING_PCM_16BIT, AudioFormat.ENCODING_AC3, AudioFormat.ENCODING_E_AC3
-          },
-          DEFAULT_MAX_CHANNEL_COUNT);
+  /** Encodings supported when the device specifies external surround sound. */
+  private static final int[] EXTERNAL_SURROUND_SOUND_CAPABILITIES =
```

Comment:
Since it is just an array of encodings, could we change it to `EXTERNAL_SURROUND_SOUND_ENCODINGS`?

Reply:
@tianyif Done. Please see 8072d63acd4cbbe6666c68f5865bfb439fbeb23c Thanks.